source/github-helpers/get-pr-info.ts

```diff
@@ -8,15 +8,27 @@ type PullRequestInfo = {
 	// https://docs.github.com/en/graphql/reference/enums#mergeablestate
 	mergeable: 'CONFLICTING' | 'MERGEABLE' | 'UNKNOWN';
 	viewerCanEditFiles: boolean;
+	headRef: {
+		compare: {
+			status: 'BEHIND' | 'DIVERGED' | 'AHEAD' | 'IDENTICAL';
+		};
+	};
 };
 
-export default async function getPrInfo(number = getConversationNumber()!): Promise<PullRequestInfo> {
+export default async function getPrInfo(head: string, number = getConversationNumber()!): Promise<PullRequestInfo> {
 	const {repository} = await api.v4(`
 		repository() {
 			pullRequest(number: ${number}) {
 				baseRefOid
 				mergeable
 				viewerCanEditFiles
+				headRef {
```

Comment:
@fregante any reason I should make a new function? This function is not used anywhere else.